src/regressor.rs

```diff
@@ -20,7 +20,7 @@ use crate::graph;
 use crate::model_instance;
 use crate::port_buffer;
 
-pub const FFM_CONTRA_BUF_LEN: usize = 20736;
+pub const FFM_CONTRA_BUF_LEN: usize = 41472;
```

Comment:
We aren't reaching any stack size limit with this right?

Reply:
Tests revealed no such issue as of yet